integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -866,6 +867,37 @@ case class CarbonLoadDataCommand(
           case _ =>
         }
       }
+
+      // Prepriming for Partition table here
+      val indexServerEnabled = CarbonProperties.getInstance().isDistributedPruningEnabled(
+        carbonLoadModel.getDatabaseName, carbonLoadModel.getTableName)
+      val prePrimingEnabled = CarbonProperties.getInstance().isIndexServerPrePrimingEnabled()
```

Comment:
`indexServerEnabled ` and `prePrimingEnabled ` looks common in many  places, better to add these two in utility and call the funtion directly inside if condition place